sdk/storage/storage-file/BreakingChanges.md

```diff
@@ -1,6 +1,12 @@
 # Breaking Changes
 
-2019.07 Version 12.0.0-preview.1
+### 2019.08
```

Comment:
Should this be a preview version?

Reply:
I thought I'd be updating these at the time of release after we are done with all the changes.
What say?

Reply:
Well, but the new version will be 12.0.0-preview.2 right? Just seems like it'd be easy to forget to change these during a future release

Reply:
I agree with @chradek

Reply:
If you want to indicate that this is not yet released, then I would suggest to skip the date, but keep the version. Something like

`Coming Soon... 12.0.0-preview.2`

And then update the date when we are closer to the release